bin/make_monthly_plot

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import os
+from desisurvey import forecast
+from importlib import reload
+import matplotlib.pyplot as plt
+
+from desiutil.log import get_logger
+log = get_logger()
+
+from argparse import ArgumentParser
+ap = ArgumentParser(description=("Generates monthly status plot. For example:"
+                    "make_monthly_plot 20240725 20240826 $SCRATCH")
+                    )
+ap.add_argument("nightbegin",
+                help=("First night of interest (YYYYMMDD). Typically a month "
+                "before nightend.")
+                )
+ap.add_argument("nightend",
+                help=("Final night of interest (YYYYMMDD). Typically today.")
+                )
+ap.add_argument("desisurveyoutput",
+                help=("Directory to write plots and ephemeris files")
+                )
+
+ns = ap.parse_args()
+
+# ADM $SURVEYOPS environment variable must be defined.
+surveyopsdir = os.environ.get('SURVEYOPS')
+if surveyopsdir is None:
+    msg = '$SURVEYOPS environment variable is not defined!'
+    log.critical(msg)
+    raise NameError(msg)
+
+log.info('Updating $SURVEYOPS/ops directory...')
+_ = os.system("svn up $SURVEYOPS/ops")
+
+os.environ['DESISURVEY_OUTPUT'] = ns.desisurveyoutput
+
+plt.clf()
+
+reload(forecast)
```

Comment:
You don't need this.  My snippet includes it because I often am changing code and then reusing it in the same ipython session without restarting, and this triggers the code to be reloaded to include my edits.

Reply:
Thanks! Good point. I've removed the statements related to the `reload`.